apps/api/src/server/app/lpa-questionnaire/lpa-questionnaire.get-appeals.routes.js

```diff
@@ -0,0 +1,10 @@
+import express from 'express';
+import { getAppeals } from './lpa-questionnaire.get-appeals.controller.js';
+
+const router = express.Router();
+
+router.get('/', getAppeals);
+
+export {
+	router as AppealWithQUestionnaireRoutes
```

Comment:
convention is to start names with capital letters only for things like classes, so could we please rename this to `appealWithQUestionnaireRoutes`?

Reply:
done!